package.json

```diff
@@ -15,7 +15,9 @@
   },
   "devDependencies": {
     "bower": "*",
+    "casperjs": "^1.1.3",
     "less": "~2",
+    "phantomjs": "^2.1.7",
```

Comment:
installing phantomjs produces a warning that the package has been renamed 'phantomjs-prebuilt'